app/src/main/kotlin/app/vitune/android/ui/screens/home/HomeDiscovery.kt

```diff
@@ -121,6 +130,7 @@ fun HomeDiscovery(
             }
         )
         val itemWidth = maxWidth * widthFactor
+        val fullWidth = maxWidth
```

Comment:
This seems useless? This should be inlined.

Reply:
I can't just inline it in like this
```kotlin
modifier = Modifier
            .width(maxWidth)
            .padding(4.dp)
```
Because maxWidth is in the scope of 'BoxWithConstraints' not in the 'RetrospectiveButton' context and it gives me this error: _'val maxWidth: Dp' can't be called in this context by implicit receiver. Use the explicit one if necessary_

If you have a better solution, let me know!